tox.ini

```diff
@@ -1,9 +1,43 @@
 [tox]
 envlist = {dev,jewel,luminous,mimic,rhcs}-{xenial_cluster,centos7_cluster,docker_cluster,update_cluster,cluster,update_docker_cluster,switch_to_containers,purge_filestore_osds_container,purge_filestore_osds_non_container,purge_cluster_non_container,purge_cluster_container,ooo_collocation}
   {dev,luminous,mimic,rhcs}-{filestore_osds_non_container,filestore_osds_container,bluestore_osds_container,bluestore_osds_non_container,bluestore_lvm_osds,lvm_osds,purge_lvm_osds,shrink_mon,shrink_osd,shrink_mon_container,shrink_osd_container,docker_cluster_collocation,purge_bluestore_osds_non_container,purge_bluestore_osds_container,lvm_batch}
+  infra_lv_create
 
 skipsdist = True
 
+# a test scenario for the lv-create.yml and lv-teardown playbooks
+[testenv:infra_lv_create]
+whitelist_externals =
+    vagrant
+    bash
+    cp
+    mkdir
+    cat
+passenv=*
+setenv=
+  ANSIBLE_SSH_ARGS = -F {changedir}/vagrant_ssh_config
+  ANSIBLE_CONFIG = -F {toxinidir}/ansible.cfg
+  ANSIBLE_ACTION_PLUGINS = {toxinidir}/plugins/actions
+  ANSIBLE_CALLBACK_PLUGINS = {toxinidir}/plugins/callback
+  ANSIBLE_CALLBACK_WHITELIST = profile_tasks
+  # only available for ansible >= 2.2
+  ANSIBLE_STDOUT_CALLBACK = debug
+deps= -r{toxinidir}/tests/requirements.txt
+changedir={toxinidir}/tests/functional/centos/7/infra_lv_create
+commands=
+  vagrant up --no-provision {posargs:--provider=virtualbox}
+  bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
+
+  cp {toxinidir}/infrastructure-playbooks/lv-create.yml {toxinidir}/lv-create.yml
```

Comment:
@andrewschoen shouldn't we copy the var file too? Because of https://github.com/ceph/ceph-ansible/pull/2922/files#diff-1bbe7aa7b5ed53b2d4e3ea99c063c0aeR27 the playbook won't find it.

Reply:
Oh I see you copied everything in `tests/functional/centos/7/infra_lv_create/group_vars/all`. You can ignore my question :).